lib/github-auth.js

```diff
@@ -2,14 +2,15 @@ var querystring = require('querystring');
 var request = require('request');
 var autosave = require('json-autosave');
 var serverSecrets;
+var baseUrl = process.env.BASE_URL || "https://shields.io";
```

Comment:
Fair point. Is it ok for you to rename all to `BASE_URL` then? This covers both use cases: the base is where the homepage is, but it's also the prefix for internal redirects etc.